wagtailio/static/sass/components/_card-grid.scss

```diff
@@ -0,0 +1,22 @@
+.card-grid {
+    margin-bottom: 20px;
+    display: flex;
+    flex-wrap: wrap;
+
+    @include media-query(medium) {
+        margin-bottom: 50px;
```

Comment:
Suggestion: there's a `sf-spacing` mixin you can use here - https://github.com/wagtail/wagtail.org/blob/next/wagtailio/static/sass/abstracts/_mixins.scss#L33